Lombiq.Tests.UI/Services/AtataScope.cs

```diff
@@ -10,14 +10,26 @@ namespace Lombiq.Tests.UI.Services;
 /// </summary>
 public sealed class AtataScope : IDisposable
 {
+    private Uri _baseUri;
+
     public AtataContext AtataContext { get; }
     public IWebDriver Driver => AtataContext.Driver;
-    public Uri BaseUri { get; }
+
+    public Uri BaseUri
+    {
+        get => _baseUri;
+
+        set
+        {
+            _baseUri = value;
+            AtataContext.BaseUrl = value.ToString();
```

Comment:
This will throw NRE if `value is null`. If this is intentional, then add `ArgumentNullException.ThrowIfNull(value);` at the beginning of the setter.